.env.template

```diff
@@ -7,9 +7,10 @@ SECRET_KEY=secret-key
 
 API_KEY=
 API_TOKEN=
-TRELLO_BOARD_ID=
-TRELLO_TODO_LIST_ID=
-TRELLO_DONE_LIST_ID=
 AZURE_CLIENT_ID=
 AZURE_TENANT_ID=
 AZURE_SUBSCRIPTION_ID=
+DOCKER_TOKEN=
+DOCKER_USERNAME=
```

Comment:
Are these actually relevant for running the app locally? If not this isn't really the right place for them (even if tracking them in your personal `.env` file is a reasonable solution!) since it's not that relevant to others